docs/openapi.yaml

```diff
@@ -1772,8 +1774,8 @@ paths:
           required: true
           schema:
             type: string
-            enum: [testnet, mainnet]
```

Comment:
I think this enum needs to stay to specify which values are allowed

Reply:
sounds good. Adding it back.